src/useCookie/__docs__/story.mdx

```diff
@@ -25,26 +34,33 @@ Manages a single cookie.
 ## Reference
 
 ```ts
+export type IUseCookieOptions = Cookies.CookieAttributes & {
+  initializeWithValue?: boolean;
+};
+
 export type IUseCookieReturn = [
   value: undefined | null | string,
   set: (value: string) => void,
   remove: () => void,
   fetch: () => void
 ];
 
-export function useCookie(key: string, options?: Cookies.CookieAttributes): IUseCookieReturn;
+export function useCookie(key: string, options: IUseCookieOptions = {}): IUseCookieReturn;
 ```
 
 #### Arguments
 
 - **key** _`string`_ - Cookie name to manage.
-- **options** _`Cookies.CookieAttributes`_ _(default: undefined)_ - Cookie options that will be
-  used during cookie set and delete.
+- **options** _`IUseCookieOptions`_ _(default: {})_ - Cookie options that will be
+  used during cookie set and delete. Has only one extra option, that relates to the hook itself:
+  - **initializeWithValue** _`boolean`_ _(default: undefined)_ - Whether to initialize state with
+    cookie value or initialize with `undefined` state.  
```

Comment:
Shouldn't the default be `true`, not `undefined`?

Reply:
In code it is undefined =)

Reply:
Hm... that doesn't really line up with our other docs for hooks with `storeDefaultValue` though.

Reply:
Where?
> While using SSR, to avoid hydration mismatch, consider setting `initializeWithValue` option
> to `false`, this will yield `undefined` state on first render and defer value fetch till effects
> execution stage.

Despite from that - maybe it is slightly confusing, as `false` and `undefined` both perceived as falsy values.

Reply:
On https://react-hookz.github.io/web/?path=/docs/side-effect-usesessionstoragevalue--example

![image](https://user-images.githubusercontent.com/6749768/121930933-f778ec00-cd10-11eb-8b34-574bca53a18b.png)


On https://react-hookz.github.io/web/?path=/docs/side-effect-usecookie--example

![image](https://user-images.githubusercontent.com/6749768/121930976-05c70800-cd11-11eb-8c6b-64ede04831e7.png)


It probably doesn't matter

Reply:
Watched the code - it defaults to `true` %)
smth with my memory today 😑